src/services/dataContext.js

```diff
@@ -19,8 +19,8 @@ export class DataContext {
     constructor(dataVersionKey, fetchDataFunction) {
         this.dataVersionKey = dataVersionKey;
         this.fetchDataFunction = fetchDataFunction;
-        this.localStorageKey = `dataContext_${this.dataVersionKey}`;
-        this.localStorageLastUpdatedKey = `dataContext_${this.dataVersionKey}_lastUpdated`;
+        this.localStorageKey = `dataContext_${this.dataVersionKey.toString()}`;
```

Comment:
had to cast to string to get rid of a warning "The value for key "dataContext_1_lastUpdated" is not a string. This can lead to unexpected behavior/errors. Consider stringifying it."